app/services/partners/family_request_create_service.rb

```diff
@@ -8,7 +8,7 @@ class FamilyRequestCreateService
 
     attr_reader :partner_user_id, :comments, :family_requests_attributes, :partner_request
 
-    def initialize(partner_user_id:, family_requests_attributes:, comments: nil, for_families: false)
+    def initialize(request_type:, partner_user_id:, family_requests_attributes:, comments: nil, for_families: false)
```

Comment:
Wait, why was this put back? We agreed that it wouldn't be needed because the service itself knows what type it is just by the current parameters being passed in.